SPECS/less/less.spec

```diff
@@ -31,6 +32,10 @@ The Less package contains a text file viewer
 %{_mandir}/*/*
 
 %changelog
+* Fri Feb 17 2023 Suresh Thelkar <sthelkar@microsoft.com> - 590-2
+- Patch CVE-2022-46663
+- License verified.
```

Comment:
It is not needed as license is verified already on line 41
```suggestion
```

Reply:
Thanks @mfrw for the review and approval. I have removed this statement as well in the latest commit.